chainer/functions/normalization/batch_normalization.py

```diff
@@ -279,7 +318,9 @@ def forward(self, inputs):
             gy = cuda.cupy.ascontiguousarray(gy)
             dtype = x.dtype
             handle = cudnn.get_handle()
-            x_desc = cudnn.create_tensor_descriptor(_as4darray(x))
+            x_desc = cudnn.create_tensor_descriptor(
+                _as4darray(x, self.key_axis))
+            cudnn_mode = self.mode.get_cudnn_mode()
```

Comment:
`cudnn_mode` is available 8 lines above.  Did you intend to move the line here?

Reply:
I will remove the one at line 315. Probably, this was back when I merged the main branch to resolve lots of conflicts. Yes, I intentionally moved it here, so that cudnn_mode is obtained at the same place as the others.